src/Queue/Sqs.php

```diff
@@ -194,6 +194,23 @@ public function deleteMessage($queueConfigName, $receiptHandle)
     }
 
     /**
+     * Purges all the messages from the queue
+     *
+     * @param string $queueConfigName
+     *
+     * @return void
+     */
+    public function purge($queueConfigName)
```

Comment:
Please add a million comments with bold and capital and flashing letters, that this can only be used in a test.... To be honest, I'd be much more happy if we just added a getter for getting the sqs client and using it to purge the queue, then to add an easy to use method for that, or it may be abused, and it can cause big problems if someone purges the wrong queue.